simulator/datastore.go

```diff
@@ -65,19 +66,15 @@ func parseDatastorePath(dsPath string) (*object.DatastorePath, types.BaseMethodF
 func (ds *Datastore) RefreshDatastore(*types.RefreshDatastore) soap.HasFault {
 	r := &methods.RefreshDatastoreBody{}
 
-	err := ds.stat()
+	_, err := os.Stat(ds.Info.GetDatastoreInfo().Url)
 	if err != nil {
 		r.Fault_ = Fault(err.Error(), &types.HostConfigFault{})
 		return r
 	}
 
 	info := ds.Info.GetDatastoreInfo()
 
-	now := time.Now()
-
-	info.Timestamp = &now
-	info.MaxMemoryFileSize = info.FreeSpace
-	info.MaxFileSize = info.FreeSpace
+	info.Timestamp = types.NewTime(time.Now())
```

Comment:
For testing (perhaps in future) and deterministic behavior, should we make time mockable, e.g. via global timeFn or a clock (https://github.com/benbjohnson/clock)?

Reply:
Not sure we need testing of the time value in this case, other than just `Timestamp != nil`

Reply:
Fine with me.